### ISSUE TITLE
Unknown message error

### ISSUE THREAD

**Issue by tarolling** [tarolling/rps-bot-old#72]
error log
```shell
COMMAND: lb by free309 (593750075763589120 - Guild: 1225118532128346193) Locale: en-US, msecs: 751
DiscordAPIError[10008]: Unknown Message
    at handleErrors (/home/minecraft/multicraft/servers/server776960/node_modules/@discordjs/rest/dist/index.js:730:13)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
    at async SequentialHandler.runRequest (/home/minecraft/multicraft/servers/server776960/node_modules/@discordjs/rest/dist/index.js:1133:23)
    at async SequentialHandler.queueRequest (/home/minecraft/multicraft/servers/server776960/node_modules/@discordjs/rest/dist/index.js:963:14)
    at async _REST.request (/home/minecraft/multicraft/servers/server776960/node_modules/@discordjs/rest/dist/index.js:1278:22)
    at async InteractionWebhook.editMessage (/home/minecraft/multicraft/servers/server776960/node_modules/discord.js/src/structures/Webhook.js:341:15)
    at async ChatInputCommandInteraction.editReply (/home/minecraft/multicraft/servers/server776960/node_modules/discord.js/src/structures/interfaces/InteractionResponses.js:161:17) {
  requestBody: {
    files: [],
    json: {
      content: undefined,
      tts: false,
      nonce: undefined,
      enforce_nonce: false,
      embeds: undefined,
      components: [],
      username: undefined,
      avatar_url: undefined,
      allowed_mentions: undefined,
      flags: undefined,
      message_reference: undefined,
      attachments: undefined,
      sticker_ids: undefined,
      thread_name: undefined,
      applied_tags: undefined,
      poll: undefined
    }
  },
  rawError: { message: 'Unknown Message', code: 10008 },
  code: 10008,
  status: 404,
  method: 'PATCH',
  url: 'https://discord.com/api/v10/webhooks/638351152131604521/aW50ZXJhY3Rpb246MTI2MzE1NDUxNjIzMzAzMTc2MDpHckRaMDZBMFJtanBsdFFzcFROUWRlSGZ2UGdrUzhmY2pOU0RHMkVRMTM4OTJDNmRQRlJVdlN5cFZGcnUxR1ZTbVVYelV4RmdnTUlDRzBBczRhUVRua3JWVGJuYU5pN2JKTTFUUTlWYlZPaXdwWGMzdUNpN0VTY1FNVEtWY05zcA/messages/@original'
}
COMMAND: stats by free309 (593750075763589120 - Guild: 1225118532128346193) Locale: en-US, msecs: 821
```

**Comment by tarolling** [comment]
Another related error
```shell
COMMAND: play by djvm (765947707411005530 - Guild: 1263236067494989886) Locale: en-US, msecs: 190
DiscordAPIError[50027]: Invalid Webhook Token
    at handleErrors (/home/minecraft/multicraft/servers/server776960/node_modules/@discordjs/rest/dist/index.js:730:13)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
    at async SequentialHandler.runRequest (/home/minecraft/multicraft/servers/server776960/node_modules/@discordjs/rest/dist/index.js:1133:23)
    at async SequentialHandler.queueRequest (/home/minecraft/multicraft/servers/server776960/node_modules/@discordjs/rest/dist/index.js:963:14)
    at async _REST.request (/home/minecraft/multicraft/servers/server776960/node_modules/@discordjs/rest/dist/index.js:1278:22)
    at async InteractionWebhook.send (/home/minecraft/multicraft/servers/server776960/node_modules/discord.js/src/structures/Webhook.js:224:15) {
  requestBody: {
    files: [],
    json: {
      content: 'Challenge timed out.',
      tts: false,
      nonce: undefined,
      enforce_nonce: false,
      embeds: undefined,
      components: undefined,
      username: undefined,
      avatar_url: undefined,
      allowed_mentions: [Object],
      flags: 64,
      message_reference: undefined,
      attachments: undefined,
      sticker_ids: undefined,
      thread_name: undefined,
      applied_tags: undefined,
      poll: undefined
    }
  },
  rawError: { message: 'Invalid Webhook Token', code: 50027 },
  code: 50027,
  status: 401,
  method: 'POST',
  url: 'https://discord.com/api/v10/webhooks/638351152131604521/aW50ZXJhY3Rpb246MTI2MzI0ODc3OTY5MjM0NzU0NDpoQjhzMmZQQ3dKNFlzcXVIaldHN2ZzbkhPbXhGcHpFajczb01YWE5KNjRpWTNlSDY5NElRRzRyMEFGUFA4aExQZlZVYlI3WkRIampwZ0l3ekFBNGwyUFo3M05ocm5hVm1DUVJzZURKNnpMYkJSWlhyTXV3MTN6ZHFwSFVsbW1YVQ?wait=true'
}
COMMAND: club by doshaer (1246527442814767158 - Guild: 1263130731958374501) Locale: pl, msecs: 646
```

**Comment by tarolling** [comment]
above patched by 8c6f78f40d7bd4b3a8d9f97adcf4be4737237e55
